app/javascript/components/components/Header.js

```diff
@@ -17,19 +17,33 @@ import { NavLink } from 'react-router-dom'
                 <p>This is the header</p>
                 <nav>
                     <ul>
-                        {logged_in && <a href="/surfspotindex">
-                            Surf Breaks
-                        </a>}
-                        {logged_in && <a href="/mycollectionsindex">
-                            My Collections
-                        </a>}
-                        {logged_in && <a href={sign_out_route}>
-                            Sign Out
-                        </a>}
+                        <NavLink to="/">
+                            Home
+                        </NavLink>
+                        <NavLink to="/aboutus">
+                            About Us
+                        </NavLink> 
+                        {logged_in &&
+                        <>
+                            <NavLink to="/surfspotindex">
+                                Surf Spots
+                            </NavLink>
+                            <NavLink to="/mycollectionsindex">
+                                My Collections
+                            </NavLink>
+                            <NavLink to="/collectionnew">
+                                Create a collection
+                            </NavLink>
+                            <NavLink to={sign_out_route}>
```

Comment:
The other links look great, but you will actually need hrefs for the devise routes.

Reply:
Got it!